src/models/endpoint/participantEndpoint.js

```diff
@@ -0,0 +1,77 @@
+/*****
```

Comment:
Please align the `model` directory `endpoint` to `participant` as per the Central-Ledger.